src/index.ts

```diff
@@ -1,70 +1,81 @@
+import { QueryType } from './models/QueryType';
+
 export default class Doxor {
+    public name;
+    public version = 1;
+
     constructor(name) {
```

Comment:
Here is a problem for when there is a database
We should handle the asynchrony in the next versions
Therefore, the constructor should remain as before

Reply:
But I checked this again
This is not a problem either

Reply:
thank you again 🙏